package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "todo",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```

Comment:
### OPTIONAL 

- Your package.json should have scripts like   `start`  to enable you to start the project with `npm start`  and enable live reloading,  please see [webpack documentation](https://webpack.js.org/guides/development/#using-webpack-dev-server) to see how to set it up  👍🏿